### ISSUE TITLE
Add an option, perhaps named `requests`, that overrides the need for using `requestsFilePath`

### ISSUE THREAD

**Issue by busterc** [hellgrenj/hulken#7]
It would be helpful to be able to simply include the requests array inside an options.json file rather than requiring a reference another requests.json file for loading at runtime.

Providing this would:
- Allow ability to add hulken as a devDependency to other modules; rather than requiring it to be installed globally
- Fix path resolution issues when using the hulken cli as a devDependency (e.g. when using `export PATH="node_modules/.bin:$PATH"`)
- Reduce file clutter

I recommend something like this:

```
if (hulken.settings.requests) {
  hulken.requestsInfo = JSON.parse(hulken.settings.requests);
  return worker._authenticateAgentsRecursive(0, worker._executeRequests);
}

fs.readFile(hulken.settings.requestsFilePath, 'utf-8', function(err, data) {
    if (err) throw err;
    hulken.requestsInfo = JSON.parse(data);
    worker._authenticateAgentsRecursive(0, worker._executeRequests);
  });
```

In place of this:
https://github.com/hellgrenj/hulken/blob/master/lib/worker.js#L16-L20

**Comment by hellgrenj** [comment]
This sounds like a good idea! unfortunately I am in a bit of a hectic period at work. I don't know when I will find time to try this out. If you like you can submit a pull request with your suggested changes. Please try to find a way to cover these changes in the tests.

**Comment by hellgrenj** [comment]
Implemented in version 1.1.0
